reader/pom.xml

```diff
@@ -31,6 +43,16 @@
             <artifactId>slf4j-log4j12</artifactId>
```

Comment:
This dependency and the jmock dependency should be declared in the dependencyManagement element of the root (parent) pom.xml. That way the versions of these components can be specified only there, and need not be specified here and in the other submodule poms.